### PR TITLE
OV 6.4 validation

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
OV 6.4 validation

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass for go 1.11 + gofmt checks.
- [x] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,59 @@
...
+- Storage System
+- Storage Template
+- Storage Volume
+- Storage Volume Attachments
+- Tasks
+- Uplink Set
+
 # [v6.3.0]
 #### Notes
 - This release extends supports of the SDK to Oneview API Version 3200.
```
Comment (AvijitOS) [inline CHANGELOG.md:59]: will this line still be there?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,20 +1,59 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
-# [v6.4.0]
+# [v6.4.0] (Unreleased)
```
Comment (AvijitOS) [inline CHANGELOG.md:3]: shall we still keep it as unreleased?

File: CHANGELOG.md (hunk 1 above)
Comment (nabhajit-ray) [reply CHANGELOG.md:59]: Yeah we keep the older releases information in the CHANGELOG

File: CHANGELOG.md (hunk 2 above)
Comment (nabhajit-ray) [reply CHANGELOG.md:3]: This will be removed during release

**Review by AvijitOS — APPROVED** [review]

**Review by chebroluharika — APPROVED** [review]

**Pull request merged**